openshift/server-with-migrations.dc.json

```diff
@@ -0,0 +1,203 @@
+{
+  "kind": "Template",
+  "apiVersion": "v1",
+  "labels": {
+    "app": "${APP_NAME}",
+    "template": "${APP_NAME}-server"
+  },
+  "parameters": [
+    {
+      "name": "NAMESPACE",
+      "description": "The OpenShift namespace in which to deploy the server",
+      "required": true
+    },
+    {
+      "name": "APP_NAME",
+      "description": "App name that OpenShift objects will be prefixed with",
+      "required": true
+    },
+    {
+      "name": "IMAGE_TAG",
+      "description": "ImageStream tag for build and deploy",
+      "required": true
+    },
+    {
+      "name": "SERVER_PORT",
+      "description": "Port the server will be exposed on",
+      "value": "5000",
+      "required": true
+    }
+  ],
+  "objects": [
+    {
+      "apiVersion": "v1",
+      "kind": "Route",
+      "metadata": {
+        "name": "${APP_NAME}-server"
+      },
+      "spec": {
+        "host": "",
+        "port": {
+          "targetPort": "web"
+        },
+        "to": {
+          "kind": "Service",
+          "name": "${APP_NAME}-server",
+          "weight": 100
+        },
+        "wildcardPolicy": "None"
+      }
+    },
+    {
+      "kind": "DeploymentConfig",
+      "apiVersion": "v1",
+      "metadata": {
+        "name": "${APP_NAME}-server"
+      },
+      "spec": {
+        "strategy": {
+          "type": "Rolling",
+          "activeDeadlineSeconds": 21600,
+          "rollingParams": {
+            "post": {
+              "execNewPod": {
+                "metadata": {
+                  "labels": {
+                    "name": "${APP_NAME}-server"
+                  }
+                },
+                "command": [
+                  "flask",
```

Comment:
we are running commands specific to our launchpad app @ashtonmeuser

Reply:
This whole file is repetition with db configs included. It will be great to move db related part as addon config file.

Reply:
Go for it. As per our discussion on Monday, I just moved our working config with DB to a new file so as not to lose a functional config. I'm game to delete this file entirely; it lives in git history as well as the Launchpad repo (currently).

Reply:
We will integrate this back to launchpad. Best thing to do is create a tech debt ticket for refactoring the config.

Reply:
Think a ticket is a good call. I'm not sure how we'll implement a server config including DB that does not repeat our standalone config and would love suggestions.